src/qtt/measurements/scans.py

```diff
@@ -849,41 +850,43 @@ def _parse_2Dvec(self):
             self['stepdata'] = stepdata
             self['sweepdata'] = sweepdata
 
-    def _convert_scanjob_vec(self, station, sweeplength=None, steplength=None, stepvalues=None):
-        """ Adjust the scanjob for vector scans. 
+    def _convert_scanjob_vec(self, station, steplength=None, sweeplength=None, stepvalues=None) -> \
+            Tuple[SweepFixedValues, SweepFixedValues]:
```

Comment:
This violation our convention to either type annotation everything or nothing at all in a single fine. But I am fine with updating our policy to allow gradual type annotation of the code.

Reply:
I added this because it confused me that the return value was sometimes a list and sometimes one instance of SweepFixedValues. I changed that to always returning a tuple. Also I changed the mixture of slicing and sweepfixedvalues. By adding the typehint I wanted to be clear what to expect from this method and the fact that I changed the signature.